CHANGELOG.md

```diff
@@ -142,6 +142,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changed: Observatory - Restored an unused particle effect for the projector
 - Changed: Observatory - The projector now loads activated instead of activating immediately upon load
 
+#### Logic Database
+
+##### Magmoor Caverns
+
+- Changed: Geothermal Core: Various Puddle Spore requirements now require Before Storage Depot B instead of After.
+
+##### Phazon Mines
+
+- Changed: Central Dynamo: Infinite Speed trick no longer requires Knowledge (Advanced)
```

Comment:
What's the reason for this? The intent of `Knowledge` wasn't for performing IS, it was for knowing that sneaking around the drone's idle pathing animation undetected to do the IS saves you several etanks of damage from the wires versus just killing the drone or being detected by it.

This is why it's 400 energy without that knowledge.

Reply:
https://discord.com/channels/914291389293027329/1127403719709962310/1233798372306194493 I never got a response for this, so I made some assumptions. I'll change it back.

Reply:
Sorry to have missed that question in that thread, all good